ISM43362/ISM43362.h

```diff
@@ -230,6 +230,8 @@ class ISM43362 {
     volatile int _active_id;
     void print_rx_buff(void);
     bool check_response(void);
+    bool check_country_code(const char* country_code);
+    char modem_country_code[5];
```

Comment:
could this be moved to the object ? 
objective is to avoid useless memory consumption

Reply:
I am not sure to understand. It is already in an C++ object.

Reply:
My mistake sorry ! this is okay